docs/mcp/client.md

```diff
@@ -25,17 +25,16 @@ Examples of both are shown below; [mcp-run-python](run-python.md) is used as the
 
 ### SSE Client
 
-[`MCPServerHTTP`][pydantic_ai.mcp.MCPServerHTTP] connects over HTTP using the [HTTP + Server Sent Events transport](https://spec.modelcontextprotocol.io/specification/2024-11-05/basic/transports/#http-with-sse) to a server.
+[`MCPServerHTTP`][pydantic_ai.mcp.MCPServerHTTP] connects over HTTP using the [Streamable HTTP transport](https://modelcontextprotocol.io/specification/2025-03-26/basic/transports) to a server.
 
 !!! note
     [`MCPServerHTTP`][pydantic_ai.mcp.MCPServerHTTP] requires an MCP server to be running and accepting HTTP connections before calling [`agent.run_mcp_servers()`][pydantic_ai.Agent.run_mcp_servers]. Running the server is not managed by PydanticAI.
 
-The name "HTTP" is used since this implemented will be adapted in future to use the new
-[Streamable HTTP](https://github.com/modelcontextprotocol/specification/pull/206) currently in development.
+The StreamableHTTP Transport is able to connect to both stateless http and sse servers.
 
-Before creating the SSE client, we need to run the server (docs [here](run-python.md)):
+Before creating the HTTP client, we need to run the server (docs [here](run-python.md)):
 
-```bash {title="terminal (run sse server)"}
+```bash {title="terminal (run http server)"}
 deno run \
   -N -R=node_modules -W=node_modules --node-modules-dir=auto \
   jsr:@pydantic/mcp-run-python sse
```

Comment:
It's a little unfortunate this is still an SSE server rather then Streamable HTTP, but I'm not going to ask you to update mcp-run-python as well :) However, can you please update note `1` below to state the URL typically ends in `/mcp` for Streamable HTTP servers, and `/sse` for Server Side Event servers?